GoogleUtilities/AppDelegateSwizzler/GULAppDelegateSwizzler.m

```diff
@@ -1002,6 +1064,89 @@ + (void)proxyAppDelegate:(id<GULApplicationDelegate>)appDelegate {
   }
 }
 
+#if UISCENE_SUPPORTED
++ (void)proxySceneDelegate:(UIScene *)scene {
+  Class realClass = [scene.delegate class];
+
+  // Skip proxying if the class has a prefix of kGULAppDelegatePrefix, which means it has been
+  // proxied before.
+  if ([NSStringFromClass(realClass) hasPrefix:kGULAppDelegatePrefix]) {
```

Comment:
nit: `NSStringFromClass(realClass)` is used multiple times. Can we assign the func result to a variable?

Reply:
Good idea, done.